lang/image.bzl

```diff
@@ -18,13 +18,30 @@ load(
     "//container:container.bzl",
     _container = "container",
 )
+load(
+    "//container:layer_tools.bzl",
+    _get_layers = "get_from_target",
+)
+
+FilterAspectInfo = provider(
```

Comment:
Do you think these should be in container/providers.bzl?

Reply:
Yes! Done.